frontend/src/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC, useContext } from 'react'
+import AuthContext from '../auth/AuthContext'
+import styled from 'styled-components/macro'
+
+const Header: FC = () => {
+  const { authUser, logout } = useContext(AuthContext)
+  return (
+    <header>
+      <h3>Project Planner</h3>
+      {authUser && logout && (
+        <section>
+          {'Sie sind eingeloggt als, '}
+          <span>{authUser.loginName}</span>
```

Comment:
```suggestion
          {'Sie sind eingeloggt als, ${authUser.loginName}'}      
```

Kannst dir die extra span wahrscheinlich sparen

Reply:
stimmt danke!